README.md

```diff
@@ -1 +1,41 @@
-Simple README for deploying KMS
+# KMS 2.0
+
+Keyword Management System (KMS) is a application for maintaining keywords (science keywords, platforms, instruments, data centers, locations, projects, services, resolution, etc.) in the GCMD/IDN system.
```

Comment:
is that true, GCMD/IDN system?

Reply:
I'll check with Scott/Tyler, not sure, took that off web page